packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -64,6 +64,29 @@ export interface CallCompositeProps extends BaseCompositeProps<CallCompositeIcon
   role?: Role;
 }
 
+/* @conditional-compile-remove(call-readiness) */
+/**
+ * Permission options
+ *
+ * @beta
+ */
+export interface DevicePermissionPrompts {
+  /**
+   * Camera Permission prompts for your call.
+   * 'required' - requires the permission to be allowed before permitting the user join the call.
+   * 'optional' - permission can be disallowed and the user is still permitted to join the call.
+   * 'doNotPrompt' - permission is not required and the user is not prompted to allow the permission.
```

Comment:
Question about `doNotPrompt`, what happens with the Calling SDK here? it will still ask for permissions so the browser window will still show if not blocked by the OS. does this just hide the devices in the drop downs?

Reply:
The device permissions here are tied with browser prompts for Microphone and Camera.
The idea is to either have prompts show up when 'required' or 'optional' is set and to hide/do not show prompts when 'doNotPrompt' is set.

You are correct, that the Calling sdk does mandate for at least one device permission (camera or microphone). 
I will be following up during Calling Office hours to address this particular issue. 
<img width="899" alt="image" src="https://user-images.githubusercontent.com/97130533/189001908-4285730d-c424-4ed2-82f7-59e99c6c14a2.png">

Reply:
so right now the calling SDK will ask for device permissions on first load of the application, how does this play into that?
this little prompt: 
![image](https://user-images.githubusercontent.com/94866715/189199142-d420b497-4e9a-419a-ba75-7b3a43ba78ba.png)